draft-ietf-tls-ctls.md

```diff
@@ -40,12 +45,12 @@ informative:
 
 --- abstract
 
-This document specifies a "compact" version of TLS 1.3. It is
-isomorphic to TLS 1.3 but saves space by trimming obsolete material,
+This document specifies a "compact" version of TLS and DTLS. It is
+equivalent to ordinary TLS, but saves space by trimming obsolete material,
```

Comment:
Hopefully isomorphic is still fine.

Reply:
Changed to "logically isomorphic", as the "logical transcript" is the same, but there is no byte-conversion isomorphism due to the distinct Finished value.